### PR TITLE
Add HoloLens to list of PowerShell adopters

### PR DESCRIPTION
**Pull request by reynoldsbd**
# Summary

Add information to ADOPTERS.md describing how PowerShell is used for Microsoft HoloLens development.

# Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [x] None
    - **OR**
    - [ ] [Experimental feature(s) needed](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Core/About/about_Experimental_Features.md)
        - [ ] Experimental feature name(s): 
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [x] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
- **Tooling**
    - [x] I have considered the user experience from a tooling perspective and don't believe tooling will be impacted.
    - **OR**
    - [ ] I have considered the user experience from a tooling perspective and enumerated concerns in the summary. This may include:
        - Impact on [PowerShell Editor Services](https://github.com/PowerShell/PowerShellEditorServices) which is used in the [PowerShell extension](https://github.com/PowerShell/vscode-powershell) for VSCode (which runs in a different PS Host).
        - Impact on Completions (both in the console and in editors) - one of PowerShell's most powerful features.
        - Impact on [PSScriptAnalyzer](https://github.com/PowerShell/PSScriptAnalyzer) (which provides linting & formatting in the editor extensions).
        - Impact on [EditorSyntax](https://github.com/PowerShell/EditorSyntax) (which provides syntax highlighting with in VSCode, GitHub, and many other editors).

### CODE REVIEW

File: ADOPTERS.md [hunk 1]
```diff
@@ -32,4 +32,5 @@ This is a list of adopters of using PowerShell in production or in their product
   Designed to let you complete tasks that should be part of a deployment, but are not possible in an ARM template today — for example, creating a Key Vault certificate or querying an external API for a new CIDR block. 
 * [Azure Pipelines Hosted Agents](https://docs.microsoft.com/azure/devops/pipelines/agents/hosted?view=azure-devops) Windows, Ubuntu, and MacOS Agents used by Azure Pipelines customers have PowerShell pre-installed so that customers can make use of it for all their CI/CD needs.
 * [GitHub Actions Virtual-Environments for Hosted Runners](https://help.github.com/actions/reference/virtual-environments-for-github-hosted-runners) Windows, Ubuntu, and MacOS virtual environments used by customers of GitHub Actions include Powershell out of the box.
+* [Microsoft HoloLens](https://www.microsoft.com/en-us/hololens) makes extensive use of PowerShell throughout the development cycle to automate tasks such as firmware assembly and automated testing.
```
Comment (iSazonov) [inline ADOPTERS.md:35]: > en-us

Can we use generic link?

Comment (SteveL-MSFT) [reply ADOPTERS.md:35]:
```suggestion
* [Microsoft HoloLens](https://www.microsoft.com/hololens) makes extensive use of PowerShell 7+ throughout the development cycle to automate tasks such as firmware assembly and automated testing.
```

**Review by SteveL-MSFT — CHANGES_REQUESTED** [review]
URLs should not contain the locale as it's detected by the website.  Also suggest calling out PS7 specifically since HoloLens dependency is specifically PS7+

**Review by SteveL-MSFT — CHANGES_REQUESTED** [review]
You'll also need to add the word `HoloLens` to our spelling dictionary right before this line: https://github.com/PowerShell/PowerShell/blob/master/.spelling#L336

**Comment by SteveL-MSFT** [comment]
CI failure is due to https://github.com/PowerShell/PowerShell/issues/12935 and isn't blocking for this PR.

**Review by SteveL-MSFT — APPROVED** [review]

**Comment by SteveL-MSFT** [comment]
@iSazonov I think we're good to merge this one

**Comment by iSazonov** [comment]
@SteveL-MSFT I haven't permissions to force the merging. So waiting CI-macos resolving.

**Comment by SteveL-MSFT** [comment]
@iSazonov ok, I rebased this branch and that should pass CI

**Pull request merged**